proposals/0035-linalg-matrix.md

```diff
@@ -262,6 +276,26 @@ Throughout this document a matrix may be described as having a scope as
 specified by the `Scope` parameter (e.g. a matrix with `Scope == Thread` is a
 _matrix with thread scope_).
 
+Matrix storage is always opaque, the `Scope` does not directly restrict how the
+matrix is stored, it merely denotes allowed scopes of allowed data divergence.
+A matrix with thread scope must behave as if each thread has a unique copy of
+the matrix, however the storage is not required to be thread local, and an
+implementation may coalesce identical matrices across threads.
+
+#### Matrix Storage
+
+In HLSL, matrix objects are intangible objects so they do not have defined size
+or memory layout. When in use, implementations are expected to spread the
+storage of matrices across the thread-local storage for all threads in a SIMD
+unit. At the DXIL level we represent a matrix through a handle object.
+
+An A matrix is a collection of per-thread vectors representing matrix rows,
+while a B matrix is a collection of per-thread vectors representing matrix
+columns.
+
+An Accumulator matrix may be either an A matrix, or a B matrix, and it varies by

```

Comment:
One additional intrinsic that may be useful related to this is a query whether an accumulator matrix is an A or B type.